src/Bicep.Core.Samples/Files/AKS_LF/main.sourcemap.bicep

```diff
@@ -0,0 +1,87 @@
+// mandatory params
+param dnsPrefix string
+//@[11:13]     "dnsPrefix": {
+param linuxAdminUsername string
+//@[14:16]     "linuxAdminUsername": {
+param sshRSAPublicKey string
+//@[17:19]     "sshRSAPublicKey": {
+
+@secure()
+param servcePrincipalClientId string
+//@[20:22]     "servcePrincipalClientId": {
+
+@secure()
+param servicePrincipalClientSecret string
+//@[23:25]     "servicePrincipalClientSecret": {
```

Comment:
I love the visual format you've added here BTW, thanks for doing this! This will allow us to have much more confidence in the future that other unrelated PRs aren't going to regress the source mapping.